### ISSUE TITLE
Flatpak build

### ISSUE THREAD

**Issue by LeandroStanger** [tModLoader/tModLoader#821]
Please, consider supporting [Flatpak](http://docs.flatpak.org/en/latest/introduction.html#reasons-to-use-flatpak) and publishing tModLoader on [Flathub](https://github.com/flathub/flathub/wiki/App-Submission).

**Comment by direwolf420** [comment]
To elaborate on why it might be important: https://forums.terraria.org/index.php?threads/mono-for-tmodloader.87078/#post-1924277

**Comment by Jofairden** [comment]
> 
> 
> To elaborate on why it might be important: https://forums.terraria.org/index.php?threads/mono-for-tmodloader.87078/#post-1924277

Ah so, tl;dr version is some Linux distros require flatpak to install stuff? Am I correct?

**Comment by LeandroStanger** [comment]
>Ah so, tl;dr version is some Linux distros require flatpak to install stuff? Am I correct?

Yes
`

**Comment by Chicken-Bones** [comment]
I'm happy to support this. Only the tML server is standalone though, would flatpak work with installing to the steam directory?

I'm happy to accept a PR if you can setup some build scripts

**Comment by LeandroStanger** [comment]
```
Only the tML server is standalone though, would flatpak work with installing to the steam directory?
```

**Comment by LeandroStanger** [comment]
>Only the tML server is standalone though, would flatpak work with installing to the steam directory?

Just give permission to access the **Steam** directory. See **Gnome Games** as an example
https://github.com/flathub/org.gnome.Games/blob/39e40256c36af73876d7f94d623c92b0dac60bb4/org.gnome.Games.json#L19

**Comment by JavidPack** [comment]
Does normal Steam install work? I'll probably close this issue soon unless normal tModLoader in normal Steam can't work.

**Comment by Chicken-Bones** [comment]
Linux server installs are being addressed in #2817
